diff --git a/node_modules/react-native-http-bridge/android/build.gradle b/node_modules/react-native-http-bridge/android/build.gradle
index 8db703c..8b09d9e 100644
--- a/node_modules/react-native-http-bridge/android/build.gradle
+++ b/node_modules/react-native-http-bridge/android/build.gradle
@@ -34,7 +34,7 @@ android {
 }
 
 dependencies {
-    compile 'com.facebook.react:react-native:+'
-    compile 'com.google.android.gms:play-services-gcm:16.1.0'
-    compile 'org.nanohttpd:nanohttpd:2.3.1'
+    implementation 'com.facebook.react:react-native:+'
+    implementation 'com.google.android.gms:play-services-gcm:16.1.0'
+    implementation 'org.nanohttpd:nanohttpd:2.3.1'
 }
diff --git a/node_modules/react-native-http-bridge/android/src/main/java/me/alwx/HttpServer/Server.java b/node_modules/react-native-http-bridge/android/src/main/java/me/alwx/HttpServer/Server.java
index decf9f2..d6606fd 100644
--- a/node_modules/react-native-http-bridge/android/src/main/java/me/alwx/HttpServer/Server.java
+++ b/node_modules/react-native-http-bridge/android/src/main/java/me/alwx/HttpServer/Server.java
@@ -14,7 +14,7 @@ import java.util.Set;
 import java.util.HashMap;
 import java.util.Random;
 
-import android.support.annotation.Nullable;
+import androidx.annotation.Nullable;
 import android.util.Log;
 
 public class Server extends NanoHTTPD {
