diff --git a/node_modules/react-native-true-sight/assets/icons/crossIcon.png b/node_modules/react-native-true-sight/assets/icons/crossIcon.png
new file mode 100644
index 0000000..9969548
Binary files /dev/null and b/node_modules/react-native-true-sight/assets/icons/crossIcon.png differ
diff --git a/node_modules/react-native-true-sight/build/images/index.d.ts b/node_modules/react-native-true-sight/build/images/index.d.ts
index 14cb1a7..71a4fe9 100644
--- a/node_modules/react-native-true-sight/build/images/index.d.ts
+++ b/node_modules/react-native-true-sight/build/images/index.d.ts
@@ -1,3 +1,4 @@
 export declare const playerPause: any;
 export declare const playerPlay: any;
 export declare const playerRestart: any;
+export declare const crossIcon: any;
diff --git a/node_modules/react-native-true-sight/build/images/index.js b/node_modules/react-native-true-sight/build/images/index.js
index b1dd398..3f575b8 100644
--- a/node_modules/react-native-true-sight/build/images/index.js
+++ b/node_modules/react-native-true-sight/build/images/index.js
@@ -1,3 +1,4 @@
 export const playerPause = require('../../assets/icons/player-pause.png');
 export const playerPlay = require('../../assets/icons/player-play.png');
 export const playerRestart = require('../../assets/icons/player-restart.png');
+export const crossIcon = require('../../assets/icons/crossIcon.png');
diff --git a/node_modules/react-native-true-sight/build/modules/video-player/components/PlayerIcon.js b/node_modules/react-native-true-sight/build/modules/video-player/components/PlayerIcon.js
index e8b5459..ebb89c8 100644
--- a/node_modules/react-native-true-sight/build/modules/video-player/components/PlayerIcon.js
+++ b/node_modules/react-native-true-sight/build/modules/video-player/components/PlayerIcon.js
@@ -1,7 +1,7 @@
 import React from "react";
 import { View, TouchableOpacity, Image, StyleSheet } from "react-native";
 export const PlayerIcon = (props) => (<View style={styles.iconWrapper}>
-    <TouchableOpacity onPress={props.onPress}>
+    <TouchableOpacity onPress={props.onPress} >
       <Image resizeMode="contain" source={props.iconSource} style={[styles.icon, { tintColor: props.iconColor ?? "#fafafa" }]}/>
     </TouchableOpacity>
   </View>);
@@ -13,5 +13,6 @@ const styles = StyleSheet.create({
         margin: 15,
         width: 30,
         height: 30,
+
     },
 });
diff --git a/node_modules/react-native-true-sight/build/modules/video-player/components/VideoPlayer.d.ts b/node_modules/react-native-true-sight/build/modules/video-player/components/VideoPlayer.d.ts
index 767ce49..15f4a88 100644
--- a/node_modules/react-native-true-sight/build/modules/video-player/components/VideoPlayer.d.ts
+++ b/node_modules/react-native-true-sight/build/modules/video-player/components/VideoPlayer.d.ts
@@ -5,6 +5,7 @@ interface PlayerProps {
     children(props: InjectedPlayerProps): React.ReactNode;
     mainControl(data: InjectedControlProps): React.ReactNode;
     bottomControl(data: InjectedControlProps): React.ReactNode;
+    onClose(): void;
 }
 export declare const VideoPlayer: React.FC<PlayerProps>;
 export {};
diff --git a/node_modules/react-native-true-sight/build/modules/video-player/components/VideoPlayer.js b/node_modules/react-native-true-sight/build/modules/video-player/components/VideoPlayer.js
index a1c23f6..efed7fd 100644
--- a/node_modules/react-native-true-sight/build/modules/video-player/components/VideoPlayer.js
+++ b/node_modules/react-native-true-sight/build/modules/video-player/components/VideoPlayer.js
@@ -1,7 +1,10 @@
 import React, { useRef, useState, useEffect } from "react";
-import { TouchableWithoutFeedback, View, Animated, StyleSheet, } from "react-native";
+import { TouchableWithoutFeedback, View, Animated, StyleSheet } from "react-native";
 import { useVideoState } from "../hooks/useVideoState";
 import { PlayerLoader } from "./PlayerLoader";
+import {PlayerIcon} from './PlayerIcon';
+import { crossIcon } from "../../../images";
+
 export const VideoPlayer = (props) => {
     const playerRef = useRef(null);
     const controlsHider = useRef(0);
@@ -70,7 +73,11 @@ export const VideoPlayer = (props) => {
         <View style={styles.loaderWrapper} pointerEvents="none">
           {videoLoading ? <PlayerLoader /> : null}
         </View>
+
         <Animated.View style={[styles.controls, { opacity: controlsFadeValue }]} pointerEvents={showVideoControls ? undefined : "none"}>
+        <View style={styles.closeIconContainer}>
+         <PlayerIcon iconSource={crossIcon} onPress={props?.onClose}/>
+        </View>
           <View style={styles.middleControlsBar}>
             {props.mainControl({
         videoPaused,
@@ -137,4 +144,9 @@ const styles = StyleSheet.create({
         left: 0,
         right: 0,
     },
+    closeIconContainer:{
+        zIndex:100,
+        height:50,
+        width:50
+    }
 });
